projects/ngrx.io/content/guide/entity/adapter.md

```diff
@@ -125,53 +125,54 @@ import { EntityState, EntityAdapter, createEntityAdapter } from '@ngrx/entity';
 import { User } from '../models/user.model';
 import * as UserActions from '../actions/user.actions';
 
-export interface State extends EntityState&lt;User&gt; {
+export interface State extends EntityState<User> {
```

Comment:
Please revert these changes. We need the use the html entities because otherwise it won't render correctly.

![image](https://user-images.githubusercontent.com/28659384/58385504-bdd1cd00-7ff1-11e9-83be-59b08ecc670c.png)

Reply:
done.